apps/services/endorsements/api/src/app/modules/endorsementList/endorsementList.controller.ts

```diff
@@ -307,4 +308,23 @@ export class EndorsementListController {
       owner: user.nationalId,
     })
   }
+
+  @ApiOperation({ summary: 'Fetches owner info from national registry' })
+  @ApiOkResponse({
+    description: 'Create an endorsements list',
+    type: String,
+  })
+  @ApiParam({ name: 'listId', type: 'string' })
+  @BypassAuth()
+  @Get(':listId/ownerInfo')
+  async getOwnerInfo(
+    @Param(
+      'listId',
+      new ParseUUIDPipe({ version: '4' }),
+      EndorsementListByIdPipe,
+    )
+    endorsementList: EndorsementList,
+  ): Promise<String> {
+    return await this.endorsementListService.getOwnerInfo(endorsementList)
+  }
```

Comment:
I wonder if it might be better to implement this as an [interceptor](https://docs.nestjs.com/interceptors).
This would allow us to centralise this functionality and might better ensure consistency